docs/options.md

```diff
@@ -15146,35 +15047,6 @@ Indentation character (Supported by Pug Beautify)
 ```
 
 
-### R beautifier
```

Comment:
It looks like your documentation was deleted in the merge. This is OK.
Please re-run the documentation generation again and commit it so we can see the `formatR` and `R language` documentation. Thanks!